grammar.js

```diff
@@ -735,6 +757,31 @@ module.exports = grammar({
       ),
     ),
 
+    _argmode: $ => choice(
```

Comment:
can you add one or two test cases for this?

Reply:
done